diff --git a/node_modules/antd-mobile-v5/es/components/mask/mask.d.ts b/node_modules/antd-mobile-v5/es/components/mask/mask.d.ts
index 6b375f6..0b3d029 100644
--- a/node_modules/antd-mobile-v5/es/components/mask/mask.d.ts
+++ b/node_modules/antd-mobile-v5/es/components/mask/mask.d.ts
@@ -14,5 +14,6 @@ export declare type MaskProps = {
     afterShow?: () => void;
     afterClose?: () => void;
     stopPropagation?: PropagationEvent[];
+    dissipationDuration?: number;
 } & NativeProps<'--z-index'>;
 export declare const Mask: React.FC<MaskProps>;
diff --git a/node_modules/antd-mobile-v5/es/components/mask/mask.js b/node_modules/antd-mobile-v5/es/components/mask/mask.js
index 7d9252c..b25ab44 100644
--- a/node_modules/antd-mobile-v5/es/components/mask/mask.js
+++ b/node_modules/antd-mobile-v5/es/components/mask/mask.js
@@ -22,7 +22,8 @@ var defaultProps = {
   opacity: 'default',
   disableBodyScroll: true,
   getContainer: null,
-  stopPropagation: ['click']
+  stopPropagation: ['click'],
+  dissipationDuration: 150,
 };
 export var Mask = function Mask(p) {
   var props = mergeProps(defaultProps, p);
@@ -53,7 +54,8 @@ export var Mask = function Mask(p) {
       mass: 1,
       tension: 200,
       friction: 30,
-      clamp: true
+      clamp: true,
+      duration: props.dissipationDuration,
     },
     onStart: function onStart() {
       setActive(true);
diff --git a/node_modules/antd-mobile-v5/es/components/popover/popover.js b/node_modules/antd-mobile-v5/es/components/popover/popover.js
index c45c6ab..ed991f0 100644
--- a/node_modules/antd-mobile-v5/es/components/popover/popover.js
+++ b/node_modules/antd-mobile-v5/es/components/popover/popover.js
@@ -46,17 +46,6 @@ export var Popover = /*#__PURE__*/forwardRef(function (p, ref) {
     }),
     onVisibleChange: setVisible,
     trigger: props.trigger,
-    motion: {
-      motionName: {
-        appear: enterClassName,
-        appearActive: enterClassName,
-        enter: enterClassName,
-        enterActive: enterClassName,
-        leaveActive: leaveClassName,
-        leave: leaveClassName
-      },
-      motionDeadline: 200
-    },
     overlay: /*#__PURE__*/React.createElement("div", {
       className: classPrefix + "-inner-content"
     }, props.content)
